### PR TITLE
Modify status cli command according to new status output

### PR DESCRIPTION
**Pull request by BelSasha**

### CODE REVIEW

**Comment by BelSasha** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/run-house/runhouse/951?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#951** <a href="https://app.graphite.dev/github/pr/run-house/runhouse/951?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#938** <a href="https://app.graphite.dev/github/pr/run-house/runhouse/938?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 1 other dependent PR ([#945](https://github.com/run-house/runhouse/pull/945) <a href="https://app.graphite.dev/github/pr/run-house/runhouse/945?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @BelSasha and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: tests/test_resources/test_clusters/test_cluster.py [hunk 1]
```diff
@@ -102,6 +102,20 @@ def run_node_all(cmd):
     return rh.here.run(cmd, node="all")
 
 
+def sort_env_servlet_processes(env_servlet_processes: dict):
```
Comment (jlewitt1) [inline tests/test_resources/test_clusters/test_cluster.py:105]: why do we need to sort for the tests?

File: runhouse/main.py [hunk 2]
```diff
@@ -217,35 +217,34 @@ def _print_cluster_config(cluster_config: Dict):
             )
 
 
-def _print_envs_info(
-    env_resource_mapping: Dict, env_servlet_processes: Dict, current_cluster: Cluster
-):
+def _print_envs_info(env_servlet_processes: Dict, current_cluster: Cluster):
```
Comment (jlewitt1) [inline runhouse/main.py:220]: can we be more descriptive with the dict type here? it's a little unclear to me what its supposed to be

File: tests/test_resources/test_clusters/test_cluster.py (hunk 1 above)
Comment (BelSasha) [reply tests/test_resources/test_clusters/test_cluster.py:105]: The `env_servlet_processes` that are retrieved from the status get endpoint ordered differently than the expected `env_servlet_processes`. To make the comparison between them easier (just run `get_status_data["env_servlet_processes"] == env_servlet_processes`), we need to sort the env_servlet_processes by their keys.

**Review by jlewitt1 — APPROVED** [review]

**Pull request merged**